aladdin-container.sh

```diff
@@ -115,7 +115,6 @@ function environment_init() {
 
         if $INIT; then
             kubectl create namespace --cluster $CLUSTER_NAME $NAMESPACE || true
-            _initialize_helm
```

Comment:
server side of helm is gone in helm3, so we don't need to do this

Reply:
If one does need to use helm2 for some reason, they would need to do this manually, correct? But it seems we're removing this helper function entirely.